app/utils.js

```diff
@@ -174,6 +175,11 @@ async function removeOrphanedImages() {
     })
 }
 
+function encodeString(str) {
+    if(!str) throw new Error('encodeString: Invalid Parameters: ' + str)
```

Comment:
URL encoding for article title requires different kind of encoding. This one is for sanitization of user's input.